### PR TITLE
updated links to contributing guidelines

### LINKED ISSUE #1328: [BUG] Several files point to the wrong CONTRIBUTING guidelines
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### Bug description

The contributing guidelines have been moved to the wiki pages, but several files are still telling users to take a look at the old contributing guidelines inside `CONTRIBUTING.md`. More specifically the [SVG Standards](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards) section.

This issue was noticed in [this](https://github.com/devicons/devicon/pull/1323#issuecomment-1197356802) PR comment.

### Possible fixes or solutions

1. Identify the links to `CONTRIBUTING.md` that can be replaced by the wiki link
2. Replace the links

### Additional information

This, I think highlights a bigger issue too: `CONTRIBUTING.md` is outdated, and can not be trusted as a real source of info for new contributors. 
Because of this I feel we should replace all the contents of the file, with a simple link to the wiki pages.

### PR DESCRIPTION
**Pull request by Snailedlt**
fix #1328

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

**This PR closes NONE**

## Notes

Didn't update link in index.html since it doesn't point to a specific place in CONTRIBUTING.md. For a general overview, the CONTRIBUTING.md file is still the best option imo.

### CODE REVIEW

File: .github/workflows/post_peek_screenshot.yml [hunk 1]
```diff
@@ -150,9 +150,9 @@ jobs:
 
             Make sure that:
 
-            - Your svgs are named and added correctly to the /icons folder as seen [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#orgGuidelines).
-            - Your icon information has been added to the `devicon.json` as seen [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#updateDevicon)
-            - Your PR title follows the format seen [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#overview)
+            - Your svgs are named and added correctly to the /icons folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs).
```
[the remark this answers is not in the document]
Comment (Snailedlt) [reply .github/workflows/post_peek_screenshot.yml]: Agreed, implemented

File: .github/workflows/post_peek_screenshot.yml [hunk 2]
```diff
@@ -150,9 +150,9 @@ jobs:
 
             Make sure that:
 
-            - Your svgs are named and added correctly to the /icons folder as seen [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#orgGuidelines).
-            - Your icon information has been added to the `devicon.json` as seen [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#updateDevicon)
-            - Your PR title follows the format seen [here](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#overview)
+            - Your SVGs are named and added correctly to the /icons folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs).
+            - Your icon information has been added to the `devicon.json` as seen [here](https://github.com/devicons/devicon/wiki/Updating-the-%60devicon.json%60)
+            - Your PR title follows the format seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
```
Comment (Panquesito7) [inline .github/workflows/post_peek_screenshot.yml:155]: I believe this is the correct link.
```suggestion
            - Your PR title follows the format seen [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: .github/workflows/post_peek_screenshot.yml (hunk 2 above)
Comment (Snailedlt) [reply .github/workflows/post_peek_screenshot.yml]: Ahh, yes you're right, thanks!

**Review by Panquesito7 — APPROVED** [review]
Amazing. Thank you! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**